src/lib.ts

```diff
@@ -285,7 +289,7 @@ export function haveCBBIngredients(fullCheck: boolean, verbose = false): boolean
     if (
       !get("instant_saveWileyWheyBar", false) &&
       !have($effect`Awfully Wily`) &&
-      myBasestat($stat`Mysticality`) < targetBaseMyst
+      myBasestat($stat`Mysticality`) < targetBaseMainStat
```

Comment:
Nearly there, just a few more checks to make sure everything is right (and ideally, a few runs to ensure that it is stable).
I think this should be
```ts
myBasestat(mainStat) < targetBaseMainStat
```